instrumentation/runtime-metrics/library/src/main/resources/jmx/rules/kafka-consumer.yaml

```diff
@@ -0,0 +1,80 @@
+---
+rules:
+  # Consumer metrics
+
+  - bean: kafka.consumer:client-id=*,type=consumer-coordinator-metrics
```

Comment:
The kafka consumer and producer metrics overlap with the metrics that are obtained using the [MetricsReporter](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/kafka/kafka-clients/kafka-clients-2.6/library) based instrumentation bridge. Definitely don't want two copies of the same data reported for agent users. And probably want to standardize on a recommended approach for library instrumentation users.

Reply:
Right. Are you suggesting removing Kafka (consumer and producer) metrics from the list of platforms supported by JMX Metric Insight?

Reply:
I think that avoiding competing mechanisms to instrument the same thing is a better UX for users. For the purposes of this PR, I suggest we remove the kafka consumer and producer configurations in favor of the `MetricsReporter` solution. After this PR is merged, we can debate the relative merits of the JMX and MetricsReporter solutions, and if JMX wins the day, delete the MetricsReporter solution.

Reply:
Sure, I'll remove them.